docs/html-reporter/html-reporter-setup.mdx

```diff
@@ -288,6 +291,79 @@ If you don't want to share analytics with us, you can disable it in any of the f
 -   Using environment variables: `html_reporter_yandex_metrika_enabled=false` or simply `NO_ANALYTICS=true`
 -   Using CLI arguments: `--html-reporter-yandex_metrika_enabled=false`
 
+### generateBadge
+
+<Version version="11.4.0" />
+
+A callback that is called after every test run. It receives the test result and returns a list of badges.
+
+Usage example: here we set the ticket name and URL, the branch name.
```

Comment:
lets not use the word "ticket"
In your title and url you use "issue", which is better. Lets use "issue" here instead of "ticket" for consistency